components/IscnUploadForm.vue

```diff
@@ -637,12 +754,26 @@ export default class UploadForm extends Vue {
     } catch (error) {
       // eslint-disable-next-line no-console
       console.error(error)
+      this.isOpenWarningSnackbar = true
+      this.error = (error as Error).toString()
+      this.uploadStatus = '';
+      return
     } finally {
       this.uploadStatus = '';
     }
 
     const uploadArweaveIdList = Array.from(this.sentArweaveTransactionInfo.values()).map(entry => entry.arweaveId);
-    this.$emit('submit', { fileRecords: this.fileRecords, arweaveIds: uploadArweaveIdList })
+    this.fileRecords.forEach((record: any, index:number) => {
+      if (this.sentArweaveTransactionInfo.has(record.ipfsHash)) {
+        const arweaveId = this.sentArweaveTransactionInfo.get(
+          record.ipfsHash,
+        )?.arweaveId
+        if (arweaveId) {
+          this.fileRecords[index].arweaveId = `ar://${arweaveId}`
```

Comment:
having `.arweaveId` prefixed with `ar://` seems like a bad idea, what is the usecase here?